src/item.cpp

```diff
@@ -817,8 +817,10 @@ std::string item::info( bool showtext, std::vector<iteminfo> &info ) const
 
     } else {
         const item *mod = this;
-        if( !is_gunmod() ) {
-            mod = &*gun_current_mode();
+        auto& aux = *gun_current_mode();
+        // if we have an active auxiliary gunmod display stats for this instead
+        if( aux.is_gunmod() && aux.is_gun() ) {
```

Comment:
Crashes because `aux` is null when displaying masterkey shotgun item in the wish item menu.